### PR TITLE
Leaderboard: final touches

### PR DESCRIPTION
**Pull request by NATASHA-ct**
- [ ] Added basic css styles and final touches

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -25,32 +26,26 @@
 To get a local copy up and running follow these simple example steps.
 
 ### Prerequisites
+-Github flow knowledge.
+-Basic CSS and HTML.
+-Installation of files.
 
 ### Setup
+-To clone the repository run `[https://github.com/NATASHA-ct/MY-PORTFOLIO.git]` in your terminal
```
Comment (TedLivist) [inline README.md]: - [ ] This link provided here is incorrect. I believe it should be: [https://github.com/NATASHA-ct/Leaderboard.git](https://github.com/NATASHA-ct/Leaderboard.git)

- [ ] Additionally, for the benefit of new users, after cloning the repository kindly include more specific instructions on how to run the app locally, like so:
  - run `cd Leaderboard`
  - run `npm install`
  - run `npm run build`
  - open `dist/index.html`

File: mods/addScore.js [hunk 2]
```diff
@@ -6,7 +6,9 @@ const newScore = (user, score) => {
   const player = document.createElement('li');
   player.innerHTML = `
                     <div class="playerDetails">
-                     <p class="playerName">${user}:</p>
+                     <p class="playerName">
+                     ${user}:
+                     </p>
                      <p class="playerScore">${score}</p>
                     </div>
                    `;
```
Comment (TedLivist) [inline mods/addScore.js:14]: - [ ] **[OPTIONAL]** Your app does not really work as expected. After adding my name and score, when I refresh nothing it does not appear.

  I put in my name and score | Refresh button is pressed (name does not appear)
  :---:|:---:
  natasha-1 | natasha-2

File: README.md [hunk 3]
```diff
@@ -25,32 +26,26 @@
...
-
-### Usage
-
-### Run tests
-
-### Deployment
-
+-Set up Github Actions
+-Set up linter > Lighthouse , Webhint ,Stylelint ,
+-Added Eslint
```
Comment (TedLivist) [inline README.md]: - [ ] **[OPTIONAL]** If you are looking to add bullet points to your README, simply add a space after `-`, like so

  ```
  - Set up Github Actions
  - Set up linter > Lighthouse , Webhint ,Stylelint ,
  - Added Eslint
  ```

**Review by TedLivist — CHANGES_REQUESTED** [review]
## STATUS: CHANGES REQUIRED ♻️ ♻️

Hi @NATASHA-ct,

Good job so far!
✔️ Nice styling
However, there are some issues that you still need to work on to go to the next project but you are almost there!

## Required Changes ♻️
_Check the comments under the review._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear. And please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: README.md (hunk 3 above)
Comment (NATASHA-ct) [reply README.md]: Thank you @TedLivist for this one, I didn't know how to do it.

**Review by TedLivist — APPROVED** [review]
## STATUS: APPROVED ✔️ ✔️

Hi @NATASHA-ct,

✔️ Cool styling

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear. And please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**